index.js

```diff
@@ -109,7 +132,8 @@ exports.flattenThread = function (thread) {
     addedIds.add(msg.key)
   }
   function flattenAndReorderReplies (msg) {
-    if (msg.value.content.type == 'post' && isaReplyTo(msg, thread)) {
+    if (thingsThatArePosts.indexOf(msg.value.content.type !== -1) && 
```

Comment:
oh god, sorry, this is exactly what [Array.prototype.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) is for...